### PR TITLE
HTTP: list keys support

### PR DESCRIPTION
**Pull request by robocoder**
Updated PR from #147 (CLIENTS-820) to address https://github.com/doctrine/cache/issues/206

### CODE REVIEW

**Comment by robocoder** [comment]
@christophermancini Yes, we're aware of the performance caveat for listing keys. We don't do this in production, but in our dev environment, we do use it to clean up before/after testing.

File: src/Riak/Command/Builder/ListObjects.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+namespace Basho\Riak\Command\Builder;
+
+use Basho\Riak;
+use Basho\Riak\Command;
+
+/**
+ * Used to list KV objects in Riak
+ *
+ * <code>
+ * $command = (new Command\Builder\ListObjecst($riak))
```
Comment (christophermancini) [inline src/Riak/Command/Builder/ListObjects.php]: Typo.

File: src/Riak/Command/Builder/ListObjects.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Basho\Riak\Command\Builder;
+
+use Basho\Riak;
+use Basho\Riak\Command;
+
+/**
+ * Used to list KV objects in Riak
```
Comment (christophermancini) [inline src/Riak/Command/Builder/ListObjects.php:9]: Can you add a disclaimer about the performance risks and not recommended for production use?

**Review by christophermancini — COMMENTED** [review]
Besides my other notes, only real thing left is adding tests for the new command.

**Comment by robocoder** [comment]
:tada:

**Review by christophermancini — APPROVED** [review]
Looks good, thanks @robocoder ! It might take me a few days to add support for the list keys command on the PHP PB lib.

**Pull request merged**